### PR TITLE
Correct windows.svg

### PR DESCRIPTION
**Pull request by Echolon**

### CODE REVIEW

**Comment by Echolon** [comment]
I dont know what to do about the 39.49

File: content/images/logos/windows.svg [hunk 1]
```diff
@@ -11,4 +11,4 @@
       d="m0,12.402,35.687-4.8602,0.0156,34.423-35.67,0.20313zm35.67,33.529,0.0277,34.453-35
       67-4.9041-0.002-29.78zm4.3261-39.025,47.318-6.906,0,41.527-47.318,0.37565zm47.329,39.
       49-0.0111,41.34-47.318-6.6784-0.0663-34.739z" />
-</svg
+</svg>
```
Comment (linkmauve) [inline content/images/logos/windows.svg:14]: It seems you used the exact same source as [on my website](https://pytouhou.linkmauve.fr/), but truncated the last column while doing your copy/paste.

I would recommend to leave this @d in a single line, with the complete file being:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 88 88">
 <path fill="#00adef" d="m0,12.402,35.687-4.8602,0.0156,34.423-35.67,0.20313zm35.67,33.529,0.0277,34.453-35.67-4.9041-0.002-29.78zm4.3261-39.025,47.318-6.906,0,41.527-47.318,0.37565zm47.329,39.349-0.0111,41.34-47.318-6.6784-0.0663-34.739z"/>
</svg>
```

Comment (Echolon) [reply content/images/logos/windows.svg:14]: Hi @linkmauve,

I think I dont understand the change requedt. Can you elaborate? You can contact me in the chat aswell

Comment (Devnol) [reply content/images/logos/windows.svg:14]: The </svg tag of the file was not closed. @linkmauve's request attempts to fix this issue. However, I did the same thing on my local machine and it still doesn't render properly. I would suggest you used the wikimedia Windows logo instead: https://commons.wikimedia.org/wiki/File:Windows_logo_-_2012.svg.

**Pull request closed without merging**